src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -169,7 +186,7 @@ private void OnWorkspaceChanged(object? sender, WorkspaceChangeEventArgs args)
 
                     if (currentContextDocumentId != null && currentContextDocumentId.ProjectId == args.ProjectId)
                     {
-                        StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: 0, selectedItemUpdateDelay: 0, updateUIWhenDone: true);
+                        StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: 0, selectedItemUpdateDelay: 0);
```

Comment:
this flag went away because it was only false in one place.  specifically, durign opening the dropdown, we would ask to start the model, then block on it, and have it not update the UI (Because we were doing it in the process of opening the UI).  Once we stopped doing that, all callers were passing in `true` and this could be removd.